src/views/Policies/PolicyView.vue

```diff
@@ -327,6 +338,8 @@ export default {
 
 <style lang="scss" scoped>
 .config-wrapper {
-  padding: var(--spacing-md);
+  padding-right: var(--spacing-md);
```

Comment:
Seen this padding 3 times, does it make sense to dry up?

Reply:
In CSS specifically, I prefer only setting properties that I truly want to set, so I will set three padding rules instead of using the shorthand where the forth property I don’t care for is set to 0. The primary reason for this is to avoid unintentionally overriding styles that are introduced by other rule sets or via the cascade.

For margins and paddings, this is perhaps overly cautious, but a good example is the `background` shorthand. Setting `background` instead of `background-color` sets **every single** property `background` is a shorthand for so specifying `background: tomato` sets `background-color: tomato` but also removes background images and everything else that might be set elsewhere.